examples/dashboard/.saturn/saturn.json

```diff
@@ -5,8 +5,8 @@
         "disk_space": "4Gi",
         "ssh_enabled": false
     },
+    "environment_variables": {},
     "description": "Deploy a dashboard to Saturn Cloud",
-    "title": "Deploy Dashboards",
```

Comment:
I like the idea of including the title here. We just need to change the validator on the schema.

Reply:
Yeah I took it out, thought about it, then just put it back